### DISCUSSION TITLE
Request failure due to CORS policy

### DISCUSSION THREAD

**Discussion by slaval2000** [cefsharp/CefSharp#4046] [General]
Using CefSharp version 99.2.90 , when using WPF Sample Example I'm able to do CROS domain request without errors ,
but when i using same code on my WPF application , I'm getting following error.

**Access to fetch at 'https://www.arcgis.com/sharing/rest/portals/self?f=json&culture=en-us' from origin 'https://ServerName' has been blocked by CORS policy: Request header field access-control-allow-headers is not allowed by Access-Control-Allow-Headers in preflight response.**

What I'm missing ?

**Comment by amaitland** [comment]
What example are you referring to exactly? There are multiple. 

> I'm able to do CROS domain request without errors ,

Are you testing with exactly the same set of urls?
